packages/webapp/components/FooterNavBar.tsx

```diff
@@ -30,15 +34,18 @@ import {
   NotificationTarget,
 } from '@dailydotdev/shared/src/lib/analytics';
 import { IconSize } from '@dailydotdev/shared/src/components/Icon';
+import { CreatePostButton } from '@dailydotdev/shared/src/components/post/write';
+import { LoggedUser } from '@dailydotdev/shared/src/lib/user';
 import styles from './FooterNavBar.module.css';
 
 type Tab = {
-  path: string;
-  title: string;
-  icon: (active: boolean, unread?: number) => ReactElement;
+  path?: string | ((user: LoggedUser) => string);
```

Comment:
This seems a bit specific, what prevents us from passing `user.permalink` directly do this component instead of function?

Reply:
the `tabs` constant is defined outside the component, the alternative would be to move it inside here, but wasn't sure if this is the right time to refactor it, or if we even should

Reply:
Hmm yeah, as I see it, should have been a component that can then take user from context. Won't block it though, its not that crucial.